.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+# Description of problem
+
+(replace this text with the list of steps you followed)
+
+# Expected result
+
+(replace this text with an explanation of what you thought would happen)
+
+# Actual result
+
+(replace this text with details of what actually happened)
+
+---
+
+(replace this text with the output of the `kata-collect-data.sh` script, after
+you have reviewed its content to ensure it does not contain any private
```

Comment:
you review its content to ensure it does not contain any private

Reply:
Hi @klynnrif - we could change this, but please note that we're using standard wording for all the repos and we already have this template in the following places:

- https://github.com/kata-containers/runtime/blob/master/.github/ISSUE_TEMPLATE.md
- https://github.com/kata-containers/agent/blob/master/.github/ISSUE_TEMPLATE.md

And the (still) pending:

- https://github.com/kata-containers/proxy/pull/63

Reply:
Ah! Gotcha :) Okay - I'll change my review. Thanks